### ISSUE TITLE
Invalid cursoMark when bufferedIn size exceed 8191 characters 

### ISSUE THREAD

**Issue by omario36** [vanitasvitae/pgpainless#220]
Version: pgpainless-core-0.2.14

Class: DecryptionStreamFactory.java
Line : 128

When message are too long ``` InputStream decoderStream = PGPUtil.getDecoderStream(bufferedIn); ``` return an inputStream with markpos=-1 

In the catch exception ``` bufferedIn.reset(); ``` rise an error :

```
java.io.IOException: Resetting to invalid mark
	at java.io.BufferedInputStream.reset(BufferedInputStream.java:448)
	at org.pgpainless.decryption_verification.DecryptionStreamFactory.parseOpenPGPDataAndCreateDecryptionStream(DecryptionStreamFactory.java:153)
	at org.pgpainless.decryption_verification.DecryptionStreamFactory.create(DecryptionStreamFactory.java:97)
	at org.pgpainless.decryption_verification.DecryptionBuilder$DecryptWithImpl.withOptions(DecryptionBuilder.java:42)
	at org.pgpainless.decryption_verification.cleartext_signatures.CleartextSignatureProcessor.getVerificationStream(CleartextSignatureProcessor.java:89)
	at fr.gael.datac.trace.handler.DatacTraceQuality.verifySignature(DatacTraceQuality.java:138)
	at fr.gael.datac.trace.handler.DatacTraceQualityTest.testVerifySignatureACRI(DatacTraceQualityTest.java:443)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)

**Comment by vanitasvitae** [comment]
Fixed with https://github.com/pgpainless/pgpainless/commit/16e283f3a6cc4e964c01c884fc79472132da51ef

**Comment by vanitasvitae** [comment]
I just released PGPainless 1.0.0-rc5 which should fix this issue.
